SimExLite/SourceCalculators/PhenomSourceCalculator.py

```diff
@@ -75,6 +77,10 @@ def __init__(
         )
 
     def init_parameters(self):
+        ### Does this have to be a public method?
```

Comment:
Yes, it's designed in `libpyvinyl` to be called by [the setter of parameters](https://github.com/PaNOSC-ViNYL/libpyvinyl/blob/master/libpyvinyl/BaseCalculator.py#L172)